runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -169,21 +193,58 @@ private void init()
+		add(food(6), ANTIPOISON_SUPERMIX1, ANTIPOISON_SUPERMIX2, ANTIDOTE_MIX1, ANTIDOTE_MIX2, ANTIFIRE_MIX1, ANTIFIRE_MIX2,
+			EXTENDED_ANTIFIRE_MIX1, EXTENDED_ANTIFIRE_MIX2, SUPER_ANTIFIRE_MIX1, SUPER_ANTIFIRE_MIX2, EXTENDED_SUPER_ANTIFIRE_MIX1, EXTENDED_SUPER_ANTIFIRE_MIX2);
+		add(combo(food(3), boost(ATTACK, perc(.10, 3))), ATTACK_MIX1, ATTACK_MIX2);
+		add(combo(food(3), boost(STRENGTH, perc(.10, 3))), STRENGTH_MIX1, STRENGTH_MIX2);
+		add(combo(5, food(3), heal(ATTACK, perc(.30, 10)), heal(STRENGTH, perc(.30, 10)), heal(DEFENCE, perc(.30, 10)), heal(RANGED, perc(.30, 10)), heal(MAGIC, perc(.30, 10))), RESTORE_MIX1, RESTORE_MIX2);
+		add(combo(food(6), boost(RUN_ENERGY, 10)), ENERGY_MIX1, ENERGY_MIX2);
+		add(combo(food(6), boost(DEFENCE, perc(.10, 3))), DEFENCE_MIX1, DEFENCE_MIX2);
+		add(combo(food(6), boost(AGILITY, 3)), AGILITY_MIX1, AGILITY_MIX2);
+		add(combo(2, food(6), boost(ATTACK, perc(.10, 3)), boost(STRENGTH, perc(.10, 3))), COMBAT_MIX1, COMBAT_MIX2);
+		add(new PrayerMix(), PRAYER_MIX1, PRAYER_MIX2);
+		add(combo(food(6), boost(ATTACK, perc(.15, 5))), SUPERATTACK_MIX1, SUPERATTACK_MIX2);
+		add(combo(food(6), boost(FISHING, 3)), FISHING_MIX1, FISHING_MIX2);
+		add(combo(food(6), boost(RUN_ENERGY, 20)), SUPER_ENERGY_MIX1, SUPER_ENERGY_MIX2);
+		add(combo(food(6), boost(HUNTER, 3)), HUNTING_MIX1, HUNTING_MIX2);
+		add(combo(food(6), boost(STRENGTH, perc(.15, 5))), SUPER_STR_MIX1, SUPER_STR_MIX2);
+		add(combo(food(6), boost(MAGIC, 3)), MAGIC_ESSENCE_MIX1, MAGIC_ESSENCE_MIX2);
+		add(new SuperRestoreMix(), SUPER_RESTORE_MIX1, SUPER_RESTORE_MIX2);
+		add(combo(food(6), boost(DEFENCE, perc(.15, 5))), SUPER_DEF_MIX1, SUPER_DEF_MIX2);
+		add(combo(food(6), boost(RANGED, perc(.10, 4))), RANGING_MIX1, RANGING_MIX2);
+		add(combo(food(6), boost(MAGIC, 4)), MAGIC_MIX1, MAGIC_MIX2);
+		add(combo(3, boost(ATTACK, perc(.20, 2)), boost(STRENGTH, perc(.12, 2)), heal(PRAYER, perc(.10, 0)), new BoostedStatBoost(DEFENCE, false, perc(.10, -2)), new BoostedStatBoost(HITPOINTS, false, perc(-.12, 6))), ZAMORAK_MIX1, ZAMORAK_MIX2);
+		add(new StaminaMix(), STAMINA_MIX1, STAMINA_MIX2);
+		add(combo(5, food(6), heal(PRAYER, perc(0.1, 2)), boost(MAGIC, perc(0.05, 2)), heal(ATTACK, perc(0.1, -2)), heal(STRENGTH, perc(0.1, -2)), heal(DEFENCE, perc(0.1, -2))), ANCIENT_MIX1, ANCIENT_MIX2);
 
 		// Regular overload (NMZ)
 		add(combo(5, boost(ATTACK, perc(.15, 5)), boost(STRENGTH, perc(.15, 5)), boost(DEFENCE, perc(.15, 5)), boost(RANGED, perc(.15, 5)), boost(MAGIC, perc(.15, 5)), heal(HITPOINTS, -50)), OVERLOAD_1, OVERLOAD_2, OVERLOAD_3, OVERLOAD_4);
 
 		// Bandages (Castle Wars)
 		add(new CastleWarsBandage(), BANDAGES);
 
+		// Bandages (Theatre of Blood entry mode)
+		add(combo(7, food(20), heal(PRAYER, perc(0.25, 5)), heal(RUN_ENERGY, 20), boost(ATTACK, perc(0.15, 4)), boost(STRENGTH, perc(0.15, 4)), boost(DEFENCE, perc(0.15, 4)), boost(RANGED, perc(0.1, 4)), boost(MAGIC, 4)), BANDAGES_25730);
```

Comment:
Do you have confirmation that the prayer healing is 25% + 5? The wiki has its value (unconfirmed) as 29, which would be accurate for a player with 99 prayer.

Reply:
I tried at 4 different levels, and they all fit that formula which is similar to prayer potions. I don't have 100 accounts with different levels to test, but I am fairly confidant that it is correct.